docs/installing.md

```diff
@@ -0,0 +1,80 @@
+
+## 1. Download dependencies
+
+1. Install git, git-lfs and Java. On ubuntu: `sudo apt install git git-lfs openjdk-11-jdk`
+2. Download the [Monero CLI](https://www.getmonero.org/downloads/#cli). We will need `monerod` and `monero-wallet-rpc`, which are contained in the archive.
+3. Download and install [Bitcoin-Qt](https://bitcoin.org/en/download).
+
+## 2. Build Haveno
+
+1. Download this repository. Run `git clone https://github.com/haveno-dex/haveno.git`
+2. Navigate inside the repository (`cd haveno`) and start the build: `./gradlew build`. It will take some minutes. If you know what you are doing you can modify [WalletConfig.java](core/src/main/java/bisq/core/btc/setup/WalletConfig.java) with custom settings before building.
+
+## 3. Run Monero
+
+1. Copy `monero-wallet-rpc`, which you downloaded with the monero CLI archive earlier (step 1.2) to the root of this repository. Make sure it has the right permissions (if you are in doubt, just run `chmod 777 monero-wallet-rpc`).
+2. Navigate into the folder containing `monerod` and run:
+
+```
+./monerod --stagenet --no-igd --hide-my-port --data-dir stagenet --add-exclusive-node 136.244.105.131:38080 --rpc-login superuser:abctesting123 --rpc-access-control-origins http://localhost:8080 --fixed-difficulty 1
+```
+
+Your daemon will connect to our private Monero stagenet and will be ready to be used by Haveno.
+
+## 4. Run Bitcoin
+
+Haveno still rely on Bitcoin for its infrastructure. We will remove it soon, in the meantime, to run Haveno locally, you'll need to download Bitcoin QT and run it in regtest mode. Which means you won't need to download the Bitcoin blockchain.
+
+1. Download and install [Bitcoin-Qt](https://bitcoin.org/en/download)
+2. Run Bitcoin-Qt in regtest mode, e.g.: `./Bitcoin-Qt -regtest -peerbloomfilters=1`
+3. Click on the *Console* tab and mine regtest bloks all at once. Run: `generatetoaddress 101 bcrt1q6j90vywv8x7eyevcnn2tn2wrlg3vsjlsvt46qz`
+
+## 5. Run Haveno
+
+Now that you have Both Bitcoin and Monero running, it's time to spin up your testing instance. You'll need to open 4 terminal windows: 1 for the seednode, 1 for the arbitrator/mediator, 1 for Alice (trader 1) and 1 for Bob (trader 2). Could be useful to rename the terminal, to have a better overview of what terminal belongs to whom (On Ubuntu: `echo -en "\033]0;NEW_TITLE\a"`).
+
+1. Run the seednode in one terminal:
+
+```
+./haveno-seednode --baseCurrencyNetwork=XMR_STAGENET --useLocalhostForP2P=true --useDevPrivilegeKeys=true --nodePort=2002 --appName=haveno-XMR_STAGENET_Seed_2002 --daoActivated=false
+```
+
+2. Run the Arbitrator instance:
+
+```
+./haveno-desktop --baseCurrencyNetwork=XMR_STAGENET --useLocalhostForP2P=true --useDevPrivilegeKeys=true --nodePort=4444 --appName=haveno-XMR_STAGENET_arbitrator --daoActivated=false --apiPassword=apitest --apiPort=9998
+```
+
+The (temporary) Haveno user interface will open. Once Haveno will be synced:
```

Comment:
What do you mean by "synced"?

Reply:
I used synced, even if it's not technically correct, because i thought it's a non-technical term people are familiar with. I meant that Haveno has successfully contacted the Monero RPC wallet and it's synced with it (so it will see transactions). Open to change it for a better alternative if we find one.

Reply:
I was just wondering if it means the monero wallet is synced